### PR TITLE
Close #10 Added ingesting admin users and groups

### LINKED ISSUE #10: Ingest Jamf users and groups
These are the users and groups found in the System Settings of Jamf UI, which controls access to the Jamf platform.

### PR DESCRIPTION
**Pull request by befirst**
Please, firstly look at:
- https://github.com/JupiterOne/jupiter-integration-jamf/pull/18
- https://github.com/JupiterOne/jupiter-integration-jamf/issues/12
- https://github.com/JupiterOne/jupiter-integration-jamf/issues/13

### CODE REVIEW

**Comment by aiwilliams** [comment]
@befirst, this will need to be rebased.

Also, I think we may have to remove the deprecated types deletion code, or change it somehow to be more specific to entities/relationships that are not produced by this new code, or else the new entities ingested in this PR will be deleted (namely, `jamf_user`).

What we really need is a way to rename these properties 🤔

**Comment by befirst** [comment]
@aiwilliams, in this PR in the second and in the third commit I have cherry-picked commits from the previous pr and in the last commit I already have deleted `deleteDeprecatedType` file and code relative to it. So now I can just fix conflicts and pr will be available for merging.

**Comment by befirst** [comment]
@aiwilliams, I have rebased it, can you please review it again?

File: src/converters/AccountAdminRelationshipConverter.test.ts [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  const admins: Admin[] = [
+    {
+      id: 6,
+      name: "Test Account",
+      directory_user: false,
+      full_name: "Test Account Full",
+      email: "test@test.com",
+      email_address: "test@test.com",
+      password_sha256:
+        "daca49d0c65f1dd3868df5524a0352e24ccc8ad5d28beb4405f878cb54f59208",
```
Comment (aiwilliams) [inline src/converters/AccountAdminRelationshipConverter.test.ts]: Are these actual values? Seems like they should be replaced with some fake data.

File: src/converters/AdminEntityConverter.test.ts [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+      directoryUser: false,
+      displayName: "Test Account",
+      email: "test@test.com",
+      emailAddress: "test@test.com",
+      enabled: "Enabled",
+      forcePasswordChange: false,
+      fullName: "Test Account Full",
+      name: "Test Account",
+      passwordSha256:
+        "daca49d0c65f1dd3868df5524a0352e24ccc8ad5d28beb4405f878cb54f59208",
```
Comment (aiwilliams) [inline src/converters/AdminEntityConverter.test.ts]: I think we should remove these from the entities. @erkangz, do you see any value in keeping this `passwordSha256` in J1?

Comment (erkangz) [reply src/converters/AdminEntityConverter.test.ts]: We should definitely remove those

**Pull request merged**